src/main/pages/che-7/end-user-guide/proc_creating-factories-in-the-dashboard.adoc

```diff
@@ -0,0 +1,171 @@
+// included in:
+// 
+// * assembly_automating-workspace-generation-using-factories
+
+[id="creating-factories-in-the-dashboard_{context}"]
+= Creating factories in the Dashboard
```

Comment:
not sure that this flow should be promoted in Che 7

Reply:
+1 let's not add these docs at this point